src/olympia/conf/dev/settings.py

```diff
@@ -260,3 +260,7 @@
 
 GITHUB_API_USER = env('GITHUB_API_USER')
 GITHUB_API_TOKEN = env('GITHUB_API_TOKEN')
+
+FXA_SQS_AWS_QUEUE_URL = (
+    'https://sqs.us-east-1.amazonaws.com/927034868273/'
+    'amo-account-change-dev')
```

Comment:
Can't we make this an environment variable and let ops configure the queues?

Reply:
well, no.  They're set up by the FxA people and are fixed.  See https://bugzilla.mozilla.org/show_bug.cgi?id=1419516#c4